tasks/main.yml

```diff
@@ -30,19 +30,6 @@
     mode: "{{ __kdump_conf.stat.mode | d('0644') }}"
   notify: Restart kdump
 
-- name: Get mode of /etc/sysconfig/kdump if it exists
-  stat:
-    path: /etc/sysconfig/kdump
-  register: __sysconfig_kdump
-
-- name: Generate /etc/sysconfig/kdump
-  template:
-    src: kdump.j2
-    dest: /etc/sysconfig/kdump
-    backup: true
-    mode: "{{ __sysconfig_kdump.stat.mode | d('0644') }}"
-  notify: Restart kdump
```

Comment:
Do we still need this for rhel6?  rhel7?  If so, we can put a conditional here.

Reply:
Oh, I haven't touched RHEL 6 for a long time, but fortunately, both RHEL6 and RHEL 7 versions of kexec-tools can generate this file.